<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translation</class>
 <widget class="QDialog" name="Translation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translation</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select WidgetConfigs file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEWidgetConfigFilePath">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectTranalatingFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select qt tranlation ts file that need to translate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Google tranlating file save path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEgoogletranlatedfileSavePath">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Directry to save google tranlating input file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select dir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Google Tranlating file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEGoogleTranlatingFilePath">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBselectUpdatingTsFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select ts file that need to update with tranlation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Google Tranlated file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEGoogleTranlatedFilePath">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectGoogleTranlatedFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Google tranlator tranlated file &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pBGenarateTranlateFile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To genarate Google tranlation input file click here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Genarate </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Static Tranlated File Save Path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEStaticTranlationFileSavePath">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Directry to save google tranlating input file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select dir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pBGenarateTranlateFile_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To genarate Google tranlation input file click here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Genarate </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBSelectTranalatingFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>widgetConfigFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>googleTranlatingFileSavePathSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>801</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBselectUpdatingTsFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>googleTranlatingFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBSelectGoogleTranlatedFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>googleTranlatedFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>staticTranlationFileSavePathSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBGenarateTranlateFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>googleTranlatingFileGenarateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBGenarateTranlateFile_2</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>staticTranlationFileGranarateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>widgetConfigFileSelectClicked()</slot>
  <slot>googleTranlatingFileSavePathSelectClicked()</slot>
  <slot>googleTranlatingFileSelectClicked()</slot>
  <slot>googleTranlatedFileSelectClicked()</slot>
  <slot>staticTranlationFileSavePathSelectClicked()</slot>
  <slot>googleTranlatingFileGenarateClicked()</slot>
  <slot>staticTranlationFileGranarateClicked()</slot>
 </slots>
</ui>
